packages/@lwc/engine-server/src/__tests__/fixtures/export-default-component/modules/x/cmp/cmp.js

```diff
@@ -0,0 +1,4 @@
+import { LightningElement } from 'lwc';
+
+class Component extends LightningElement {}
+export default Component;
```

Comment:
Could you please add a test case for `export { Component as default }` as well? This is the one that doesn't work currently: https://github.com/salesforce/lwc/issues/4020